src/apis/auth.ts

```diff
@@ -1,30 +1,29 @@
+import { UserProps } from '../hooks/useAuth';
 import { LoginProps } from '../pages/User/Login';
-import { SignupProps } from '../pages/User/Signup';
 import { httpClient } from './http';
 
-export const join = async (userData: SignupProps) => {
+export const join = async (userData: UserProps) => {
   try {
-    const response = await httpClient.post(`/users/join`, userData);
-    console.log(httpClient);
+    const headers = {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${userData.uid}`,
+    };
+    const body = {
+      nickname: userData.nickname,
+      image: userData.image,
+      team: userData.team,
+    };
+    const response = await httpClient.post(`/users/join`, body, {
```

Comment:
자바스크립트 표현식 없이 문자열만 작성할때는 백틱보다 작은따옴표나 큰따옴표를 사용해주세요~
```suggestion
    const response = await httpClient.post('/users/join', body, {
```